uwblueprint/sistering
# Read only schedule details page

## Ticket link

Closes #474



## Implementation description
* Add the correct getPostingStatus code (need follow up to replace incorrect usage)
  * moved code into type utils 
* Make a read only variant of the schedule posting page (now exposed to employees as well)
  * this page only shows choosen/confirmed volunteers vs available volunteers for that posting

![image](https://user-images.githubusercontent.com/44826218/179375391-3ad36065-7e63-4bd1-85c1-562bf879e0c2.png)


## Steps to test
1. Go to http://localhost:3000/admin/schedule/posting/1 in two separate browser sessions (1 admin, 1 employee)
2. Both should be able to see the page without error
3. Employee users should not be able to see any "available volunteers" but only confirmed or publish confirmed volunteers for each shift
4. Employees cannot edit these pages
5. A "past" posting, a posting with it's end date earlier than the current date will also be read only



## What should reviewers focus on?
* correctness
* any edge cases missed with any mutable actions not hidden when in read only mode


## Checklist
